tests/foreman/api/test_permission.py

```diff
@@ -249,15 +261,31 @@ def test_positive_check_create(self):
 
         :CaseImportance: Critical
         """
-        for entity_cls in (entities.Architecture, entities.Domain):
+        for entity_cls in (
+                entities.Architecture,
+                entities.Domain,
+                entities.ActivationKey):
             with self.subTest(entity_cls):
                 with self.assertRaises(HTTPError):
                     entity_cls(self.cfg).create()
                 self.give_user_permission(
                     _permission_name(entity_cls, 'create')
                 )
-                entity_id = entity_cls(self.cfg).create_json()['id']
-                entity_cls(id=entity_id).read()  # As admin user.
+                entity = entity_cls(self.cfg)
+                entity_fields = entity.get_fields()
+                if 'organization' in entity_fields:
+                    if entity_fields['organization'] is OneToManyField:
```

Comment:
Not sure about this one, but I wasn't able to find a way to distinguish between entities that expect list of organizations and the ones that expect only one organization.